app/controllers/missions_controller.rb

```diff
@@ -80,12 +71,10 @@ def order_deliverables
 
   private
 
-  # Use callbacks to share common setup or constraints between actions.
   def set_mission
     @mission = Mission.find(params[:id])
```

Comment:
That makes sense when we have a page just for the user's missions, i.e. not a generic public page.